lib/stitches/linear_gradient_fill.py

```diff
@@ -289,7 +289,7 @@ def _get_stitch_groups(fill, shape, colors, color_lines, starting_point, ending_
         graph_make_valid(fill_stitch_graph)
 
         travel_graph = build_travel_graph(fill_stitch_graph, shape, fill.angle, False)
-        path = find_stitch_path(fill_stitch_graph, travel_graph, starting_point, ending_point)
+        path = find_stitch_path(fill_stitch_graph, travel_graph, starting_point, ending_point, False)
```

Comment:
Any reason why `False` here instead of `None`?

Reply:
I definitely want underpathing off for linear gradient fills